site.js

```diff
@@ -353,6 +353,10 @@ $(document).ready(function (){
     ],
     'lib3to6': [
       { content:'Python 3 only', start: '2018-09-01', end: '2023-12-31', py2:false},
+    ],
+    'Connectome Mapper 3':[
+      {content: 'v3.0.0-beta-x beta series (Py2 only)', start: '2019-04-24', end:'2020-08-24', py2:true},
+      {content: 'v3.x.y[-RCz] releases and release candidates (Py3 only)', start: '2020-08-24', end:'2023-12-31'},
```

Comment:
This is pretty confusing. Your betas were Python 2 only but your release candidates onwards were Python 3 only? It might be better just to leave it out, or say that the 3.x series is Python 3 only.